charts/redpanda/values.yaml

```diff
@@ -403,21 +399,22 @@ tuning: {}
 # listeners.
 #
 listeners:
-  # Admin API listener
-  # The kafka listener group cannot be disabled
+  # Admin API listener (only one)
   admin:
-    # The port for the admin server
+    # The port for both internal and external connections to the admin API
```

Comment:
This cannot be true.

Reply:
There is only ever a single admin API port. Internal and external clients always connect to the same port.

Reply:
Ah, right, so there's no "advertised listeners" for admin then.